File: rpg-client-java/src/niveles/Nivel3.java
package niveles;

import utilidades.Acciones;
import static utilidades.Acciones.*;

public class Nivel3 {

	private static Acciones acciones;

	public static void iniciar() {

		acciones = new Acciones();
		acciones.start();

		reset();
	}

	public static void finalizar() {

		obtenerPosicion((x, y) -> {

			if ((x >= 65) && (x <= 210) && (y >= 128) && (y <= 278)) {
				enviarMensaje("¡Nivel superado!");
			}

			end();
		});
	}
}

File: rpg-client-typescript/src/niveles/nivel3.ts
import * as acciones from '../utilidades/acciones'


export const iniciar = () => {
  acciones.reset();
}

export const finalizar = (contexto: any) => {
  
  acciones.obtenerPosicion((x, y) => {

    if ((x >= 65) && (x <= 210) && (y >= 128) && (y <= 278)) {
      acciones.enviarMensaje('¡Nivel superado!')
    }
    
    acciones.end();
  });
}